README.md

```diff
@@ -59,9 +59,11 @@ any errors or bugs with ingestion.
 
 ## Additional References
 
+- [GUAC presentation at OSS NA 2023](https://sched.co/1K5Hn)
```

Comment:
Did you want to add in the deps.dev and OCI ingestion piece above also to this PR?

Reply:
added deps.dev and OSV , no section for collectors right now, so leaving that for different pr

Reply:
Oops I meant OSV!